packages/documentation/src/stories/utilities/sizing/sizing.stories.ts

```diff
@@ -1,15 +1,122 @@
-import { Meta, StoryObj } from '@storybook/web-components';
+import type { Args, Meta, StoryObj, StoryContext } from '@storybook/web-components';
+import { html } from 'lit/static-html.js';
```

Comment:
```suggestion
import { html } from 'lit';
```
Doesn't make a big difference, but it's more future-proof.